README.md

```diff
@@ -1,128 +1,147 @@
-print(x.upper())       # Will print string will all chars capitalised
-print(x.capitalised()) # Will print string but with first char capitalised
+print(x.upper())       # Will print string will all chars capitalized
+print(x.capitalize()) # Will print string but with first char capitalized
 ```
 
-### def
+### `def`
 
-In python you can define functions like this
-```
+In python you can define functions like this :
+
+```python
 def SomeFunction(argument1, argument2):
   temp = argument1 + argument2  # Add both arguments
   output = temp**2              # Square variable temp
   return output                 # Return the value of the variable output
 ```
-and used like this in a script
-```
+
+... and used them this way in a script :
+
+```python
 x = SomeFunction(2, 3)
 print(x) # Would return (2+3)**2 = 25
 ```
 
-### Class
+### `Class`
 
-In python you can define your own classes and inherit from other classes
-```
-class MyClass(ParentClass):                      # Leave empty of there is no need for a Parent class
+In python you can define your own classes, and inherit from other classes :
+
+```python
+class MyClass(ParentClass):  # Leave empty of there is no need for a Parent class
   def __init__(self, arg1, arg2, arg3, ...):
-    ParentClass.__init__(self, arg1, arg2, ...)
-    self.stuff = arg1
-    self.otherStuff = arg2
-    self.lastBitOfStuff = arg3
+	ParentClass.__init__(self, arg1, arg2, ...)
+	self.stuff = arg1
+	self.otherStuff = arg2
+	self.lastBitOfStuff = arg3
+
   def NewMethod(self, SomeArgument):
-    self.stuff = SomeArgument
-```
-in the code:
+	self.stuff = SomeArgument
 ```
+
+In the code :
+
+```python
 ################################################################################
-##### Definine usefull classes #################################################
+##### Definite useful classes ##################################################
 ################################################################################
 class Human():
   def __init__(self, Name, Age):
-    self.Name = Name
-    self.Age = Age
+	self.Name = Name
+	self.Age = Age
+
   def setName(self,NewName):
-    self.Name = NewName.capitalise()  # Will change my name and make sure it's 
-                                      # capitalised
-    
+  	# Will change my name and make sure it's capitalized
+	self.Name = NewName.capitalize()
+
 ################################################################################
 ##### Main script ##############################################################
 ################################################################################
 if __name__ == "__main__":
-  Moi = Human("Alex", 33)
-  print(Moi.Age)                        # Will print my age, 33
-  print(Moi.Name)                       # Will print my name, Alex
-  Moi.setName("charles")                # Will change my name to Charles
-  print(Moi.Name)                       # Will print my name, Charles
+  Me = Human("Alex", 33)
```

Comment:
Don't like french in code?

Reply:
Bien sûr que si, mais pas quand tout le reste est en anglais 😄

Reply:
C'est important de noter que ce repo est le resultat d'une séance live de code pour des amis à moi qui n'avaient jamais codé. Apres un certain temps la limite du français et l'anglais était floue! 
Mais merci pour le pull request. J'espere avoir le temps de rajouter des "module" dans le future quand mes amis vont me demander de leur montrer des nouvelles choses!

Reply:
Ah très bien ! Si tu te bases sur un service comme GitHub pour ces petites "séances", je te conseillerais de t'installer un de ses équivalents (mais _libres_), tels que [Gitea](https://gitea.io/), ou bien [NotABug](https://notabug.org/), que tu hébergerais toi-même, dans le but de garder ton profil GitHub (public !) "propre" !

👋